ItemRackOptions/ItemRackOptions.lua

```diff
@@ -452,10 +453,12 @@ function ItemRackOpt.LoadSet()
 end
 
 function ItemRackOpt.DeleteSet()
-	ItemRackUser.Sets[ItemRackOptSetsName:GetText()] = nil
+	local setname = ItemRackOptSetsName:GetText()
+	ItemRackUser.Sets[setname] = nil
 	ItemRackOpt.PopulateSetList()
 	ItemRack.CleanupEvents()
 	ItemRackOpt.PopulateEventList()
+	ItemRack:FireItemRackEvent("SET_DELETED", setname)
```

Comment:
Please call the event in a way that its name is ItemRack specific, like ITEMRACK_SET_DELETED